tests/fullstack-test/expr/regexp.test

```diff
@@ -94,3 +94,24 @@ mysql> set tidb_enforce_mpp=1; select regexp_like(data, pattern, match_type) as
 |    1 |
 |    1 |
 +------+
+
+mysql> drop table if exists test.t;
+mysql> create table test.t (data varchar(30), pattern varchar(30), pos int, occur int, ret_op int, match_type varchar(30));
+mysql> alter table test.t set tiflash replica 1;
+func> wait_table test t
+mysql> set tidb_enforce_mpp=1; select regexp_instr("1", "1", pos, occur, 1, match_type) as res from test.t;
+mysql> set tidb_enforce_mpp=1; select regexp_instr("1", "", pos, occur, 1, match_type) as res from test.t;
+
+mysql> drop table if exists test.t;
```

Comment:
Why need to drop the table and recreate a new one?

Reply:
> Why need to drop the table and recreate a new one?

I will delete it.